cafebabel/templates/articles/macros.html

```diff
@@ -22,8 +22,8 @@
     <ul class=tags-list>
       {% if article and article.tags %}
         {% for tag in article.tags -%}
-          <li class=saved>
-            {{ tag.name }}{# Avoid any whitespace between the name and the input.
+          <li class=saved>{# Avoid any whitespace before the name and the input.
```

Comment:
is it an issue also before?

Reply:
I try without, the tests pass and it seems works. So thx